test_mcp_registration.sh

```
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Test script to verify MCP tool registration
+
+# Change to src-tauri directory
+cd src-tauri
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)